lib/events/gcssessions/gcsstream.go

```diff
@@ -275,6 +276,15 @@ func (h *Handler) ListUploads(ctx context.Context) ([]events.StreamUpload, error
 	return uploads, nil
 }
 
+// GetUploadMetadata gets the metadata for session upload
+func (h *Handler) GetUploadMetadata(s session.ID) *events.UploadMetadata {
+	url := fmt.Sprintf("%v://%v/%v", teleport.SchemeGCS, h.path, string(s))
```

Comment:
i'd just use the result of `fmt.Sprintf` directly:
```go
return &events.UploadMetadata{
	URL: fmt.Sprintf("%v://%v/%v", teleport.SchemeGCS, h.path, string(s)),
	SessionID: s,
}
```
Also, the source code needs to be gofmt-ed.